simapp/app.go

```diff
@@ -176,16 +176,19 @@ func NewSimApp(
 	app.subspaces[crisis.ModuleName] = app.ParamsKeeper.Subspace(crisis.DefaultParamspace)
 	app.subspaces[evidence.ModuleName] = app.ParamsKeeper.Subspace(evidence.DefaultParamspace)
 
+	// add capability keeper and ScopeToModule for ibc module
+	app.CapabilityKeeper = capability.NewKeeper(
+		app.cdc, keys[capability.StoreKey], memKeys[capability.MemStoreKey],
+	)
+	scopedIBCKeeper := app.CapabilityKeeper.ScopeToModule("ibc")
```

Comment:
Currently using single ScopedKeeper for all of IBC, does that make sense? Or should i have seperate scoped Keepers for channelKeeper and portKeeper

Reply:
The `x/ibc` module should have a single `ScopedCapabilityKeeper`, except `20-transfer` which should have its own (probably `20-transfer` should be moved out of `x/ibc` actually).